src/scanner.c

```diff
@@ -386,6 +394,7 @@ UInt OpenInput (
 
     /* remember the current position in the current file                   */
     if ( STACK_SIZE(Input) > 0 ) {
+        GAP_ASSERT(CHAR_PUSHBACK_EMPTY());
```

Comment:
I was briefly confused by this line, then I realized `CHAR_PUSHBACK_EMPTY()` is tester function returning a boolean. So perhaps call it `IS_CHAR_PUSHBACK_EMPTY` to make it crystal clear?